### PR TITLE
feat(config): make more elements configurable and add static logo width

### PR DESCRIPTION
**Pull request by Guitarninja2**

### CODE REVIEW

File: src/components/TheHeader.vue [hunk 1]
```diff
@@ -63,6 +63,7 @@ $logoMargin: 2px;
 
 .c-header__image {
   margin-top: $logoMargin;
+  width: 265px;
```
Comment (MattLish) [inline src/components/TheHeader.vue:66]: Do we need a static image width? Ideally, I would like the launcher to be resizeable in the future

**Review by MattLish — CHANGES_REQUESTED** [review]

File: src/components/TheHeader.vue (hunk 1 above)
Comment (Guitarninja2) [reply src/components/TheHeader.vue:66]: It's to keep the logo from pushing elements out of the way and causing the launcher to overflow when you use a larger logo. I think the reason it works fine right now is the current Wildlander logo is the exact default size to make things not overflow or anything.

Comment (Guitarninja2) [reply src/components/TheHeader.vue:66]: I'm fine just making the edit on my fork though if necessary. It just may be useful for others with different sized logos.

Comment (RingComics) [reply src/components/TheHeader.vue:66]: This repo is for the Wildlander specific launcher. You may want to check out [Azura's Star](https://github.com/RingComics/azuras-star), thats where we'll be doing the modlist agnostic launcher template ~~when I get to it~~.

Closing this PR

**Pull request closed without merging**